lmfdb/ecnf/test_ecnf.py

```diff
@@ -105,6 +105,14 @@ def test_isodeg(self):
         L = self.tc.get('EllipticCurve/?start=0&torsion=1&isodeg=2')
         assert 'No matches' in L.get_data(as_text=True)
 
+    def test_cm_field_search(self):
+        r"""
+        Test that searching for CM field discriminant works
+        """
+        L = self.tc.get('/EllipticCurve/?cm_field=-4')
+        assert '1024.1-c1' in L.get_data(as_text=True)
+        assert '1.0.1-a1' not in L.get_data(as_text=True)
```

Comment:
Better to use `self.check_args('/EllipticCurve/?cm_field=-4', ['1024.1-c1', '1.0.1-a1'])`

Reply:
Oops, the ability to put a list there depends on a feature in a branch of mine that hasn't been merged yet.  You can still do it in two `check_args` statements.

Reply:
Thanks, I didn't know about `check_args` and `not_check_args`!